src/domain/fist/class.ts

```diff
@@ -0,0 +1,10 @@
+import { FrosthavenClassNames } from '../frosthaven-class.type';
+import { fistCards } from './cards';
+
+export const fist = {
+  name: FrosthavenClassNames.Fist,
+  handSize: 11,
```

Comment:
```suggestion
  handSize: 8,
```
Haven't unlocked him yet but it seems his hand size is 8